### DISCUSSION TITLE
项目还会继续开发维护么？云大博客说灵犀准备关闭这个项目？

### DISCUSSION THREAD

**Discussion by Gowa2017** [ejoy/ant#158] [General]
如题

**Comment by constly** [comment]
博客里面已经说了，会继续维护的，比如最近引擎就一直在更新~

**Reply by cloudwu** [reply]
对。我自己也要用。

这个项目已经开源，只是灵犀不投资了而已。我们所有的开发人员都在业务时间维护。

最近在探索新的合作流程。和之前坐在一个办公室不同，现在开发人员不在一起工作，所以出现问题反馈流程会长一些。
